content/applications/sales/point_of_sale/pricing/pricelists.rst

```diff
@@ -2,45 +2,118 @@
+        multiple computations:
+
+        - Define what the calculation is based on (the :guilabel:`Sales Price`, the
+          :guilabel:`Cost`, or an :guilabel:`Other Pricelist`.)
+        - Apply a :guilabel:`Discount` and/or add an :guilabel:`Extra Fee`.
+        - Define a :guilabel:`Rounding Method` and specify the minimum amount of **margin**
+          over the base price *(optional)*.
+
+      Then, select the products you want to apply this computation to:
+
+      - :guilabel:`All Products`
+      - a :guilabel:`Product Category`
+      - a specific :guilabel:`Product`
+      - a specific :guilabel:`Product Variant`
+
+      And add conditions such as a specific quantity to reach for the price to change in the
+      :guilabel:`Min. Quantity` field or a specific period of time during which the pricelist can
+      be applied in the :guilabel:`Validity` fields.
+
+      .. image:: pricelists/price-rules.png
+         :alt: setup form to configure an advanced pricelist
 
-.. image:: pricelists/pricelists_01.png
-   :align: center
+      .. note::
+         Pricelists can be selected if the conditions are unmet, but the computation will not be
+         applied.
 
-Now, you can create *Pricelists* by clicking on the *Pricelists* link.
-Then, set it up by choosing the product category you want to include in your happy hour
-and the discount.
+Once pricelists are created, go to the :ref:`POS settings <configuration/settings>` and select
+either one pricelist in the :guilabel:`Default Pricelist` field or tick the :guilabel:`Advanced
+Pricelists` checkbox and fill in the :guilabel:`Available` field with all the desired pricelists.
 
-.. image:: pricelists/pricelists_02.png
-   :align: center
+.. note::
+   All available pricelists must be in the same currency as the company or the **Sales Journal** set
+   on the selected point of sale.
 
-Go back to your *PoS* settings and add the Happy Hour pricelist to the
-list. You can even choose a default pricelist if needed.
+Use pricelists
+==============
 
-.. image:: pricelists/pricelists_03.png
-   :align: center
+:ref:`Open a POS session <pos/session-start>`, click the **pricelists** button, and select the
+desired pricelist from the list.
 
-From now on, on the *PoS* interface, a new button is available, allowing you to choose
-among the different *pricelists* you added before.
+.. image:: pricelists/pricelist-button.png
+   :alt: button to select a pricelist on the pos frontend
 
-.. image:: pricelists/pricelists_04.png
-   :align: center
+You can also set a pricelist to be selected automatically once a specific :ref:`customer is set
+<pos/customers>`. To do so, go to the customer form and switch to the preferred pricelist on the
+:guilabel:`Pricelist` field of the :guilabel:`Sales & Purchase` tab.
 
 .. seealso::
-   * :doc:`../../sales/products_prices/prices/pricing`
+   - :doc:`../../sales/products_prices/prices/pricing`
+   - :ref:`multi_website/pricelists`
```

Comment:
It might be more useful to add a label here as I feel "Pricelists" is too general (and probably confusing, especially since it sends us to the multi-website doc and not a "general" doc on pricelists). Also, I changed that title in the multi-website doc to "Pricing" in v16, which is even worse in terms of "too general" ^^